test/unit/base.test.js

```diff
@@ -2011,6 +2011,64 @@ describe('INTERFACE FUNCTION TESTS ', function () {
       });
     });
   });
+
+  describe('The converter must identify valid OA3 specification', function () {
+    var pathPrefix = VALID_OPENAPI_PATH,
+      sampleSpecs = fs.readdirSync(path.join(__dirname, pathPrefix)),
+      oa3_1_files = ['issue#479_2.yaml', 'issue#10229.json', 'query_param_with_enum_resolve_as_example.json'];
```

Comment:
@SahilChoudhary22 Let's also verfify that for these files the `specificationVersion` is 3.1.x.